src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -3,139 +3,152 @@
 
 {% block sidebar_content %}
 {% anchor_menu anchors=anchors desktop=True %}
-{% if product.link %}
+{% if product.form_snippet %}
     <li class="anchor-menu__list-item">
-        {% button href=product.link size="big" text=_("Aanvraag starten") primary=True icon="arrow_forward" icon_position="before" %}
+        {% button id="product-openforms-button" size="big" text=_("Aanvraag starten") primary=True icon="arrow_forward" icon_position="before" %}
+    </li>
+{% elif product.link %}
+    <li class="anchor-menu__list-item">
+        {% button id="product-link-buton" href=product.link size="big" text=_("Aanvraag starten") primary=True icon="arrow_forward" icon_position="before" %}
     </li>
 {% endif %}
 {% endanchor_menu %}
 {% endblock sidebar_content %}
 
 {% block content %}
-    <h1 class="h1" id="title">
-        {{ object.name }}
-        {% if request.user.is_staff %}
-            {% button icon="edit" text=_("Bewerken in de Admin") hide_text=True href="admin:pdc_product_change" object_id=object.pk %}
-        {% endif %}
-    </h1>
-    {% tag tags=object.tags.all %}
-    <p class="p">{{ object.summary }}</p>
-    {{ object.content|ckeditor_content|safe }}
-    {% if object.files.exists %}
-        <hr class="divider">
-        {% file_list files=object.files.all title=_("Bestanden") %}
+
+    {% if product.form_snippet %}
+        <div class="openforms">
+            {{ product.form_snippet|safe }}
```

Comment:
Was there no other way of rendering this without using the safe?
Also, this will not load with the CSP headers enabled.